test/moonwall.config.json

```diff
@@ -8,15 +8,18 @@
             "name": "dev_tanssi",
             "envVars": ["DEBUG_COLORS=1"],
             "testFileDir": ["suites/dev-tanssi", "suites/common"],
+            "timeout": 120000,
+            "include": ["**/*test*"],
             "multiThreads": true,
-            "reporters": ["basic"],
+            "reporters": ["basic", "html", "json"],
+            "reportFile": { "json": "./tmp/testResults.json" },
```

Comment:
This makes it a bit harder to see test failures when running locally, can we revert it for now?

Reply:
yes